client/src/components/TaskCard/TaskCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { PropTypes } from "prop-types";
+import SmallActionButton from "../SmallActionButton/SmallActionButton.js";
+import styles from "./TaskCard.module.css";
+
+const TaskCard = ({
+  title,
+  location,
+  startTime,
+  endTime,
+  isOngoing,
+  showDetailsDialog,
+}) => {
+  const [taskOngoing, setOngoing] = useState(isOngoing);
+  const taskTextStyle = !taskOngoing ? styles.taskDone : "";
+  return (
+    <div className={styles.taskContainer}>
+      <div className={styles.colorBar}></div>
+      <div className={styles.taskInfo} onClick={showDetailsDialog}>
+        <p className={taskTextStyle}>{title}</p>
+        <p className={taskTextStyle}>{location}</p>
+      </div>
+      <div className={styles.timeInfo}>
+        <p className={taskTextStyle}>{startTime}</p>
+        <p className={taskOngoing ? styles.endTimeStyle : styles.taskDone}>
```

Comment:
I noticed that the start time and end time are different font sizes when a task is started/completed. Is this the intention that's wanted for a complete task?

![image](https://user-images.githubusercontent.com/24628243/157380751-54cf06f4-2112-404b-a705-956beecf63b2.png)